### PR TITLE
doc: CI system link added, clarity increased

### PR DESCRIPTION
**Pull request by sushant-varanasi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ There are also [regression and integration tests](/test), written
 in Python, that are run automatically on the build server.
```
Comment (MarcoFalke) [inline README.md:56]:
```suggestion
in Python.
```

**Comment by practicalswift** [comment]
ACK 1a520fc8dbe32b9f9fd4b38c4b70660a64e26e05: why not :)

File: README.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ There are also [regression and integration tests](/test), written
...
 These tests can be run (if the [test dependencies](/test) are installed) with: `test/functional/test_runner.py`
 
-The Travis CI system makes sure that every pull request is built for Windows, Linux, and macOS, and that unit/sanity tests are run automatically.
+The [Travis CI system](https://travis-ci.org/) makes sure that every pull request is built for Windows, Linux, and macOS, and that unit/sanity tests are run automatically.
```
Comment (MarcoFalke) [inline README.md:59]:
```suggestion
The [CI system](/ci) defines build configs for Windows, Linux, and macOS, unit/sanity tests are part of the CI configs.
```

**Review by maflcko — COMMENTED** [review]
We've switched to our own ci system with config files. They can be executed on any machine. Currently we use Appveyor, Cirrus and Travis for build resources, but I don't think it is worth listing all of them in the top-level readme.

I suggest to strip down the verbosity a bit.

File: README.md (hunk 1 above)
Comment (sushant-varanasi) [reply README.md]: commit done

File: README.md (hunk 2 above)
Comment (sushant-varanasi) [reply README.md]: commit done

**Comment by sushant-varanasi** [comment]
@MarcoFalke @practicalswift could you merge this pr now.

**Comment by practicalswift** [comment]
@sushant-varanasi Please squash :)

**Comment by MarcoFalke** [comment]
> could you merge this pr now

This is not how contributing works here. Please refer to our docs

**Comment by sushant-varanasi** [comment]
"If you have problems with squashing or other git workflows, you can enable "Allow edits from maintainers" in the right-hand sidebar of the GitHub web interface and ask for help in the pull request."
- Sentence in the contribution docs

@practicalswift I think u can "squash and merge" as I have allowed edits by maintainers.

**Comment by fanquake** [comment]
In future, don't open multiple PRs (#20224) for the same change. If you'd like to contribute to this project you'll need to learn how to use git. That includes how to squash, rebase, edit commits messages etc.

**Pull request closed without merging**